package-lock.json

```diff
@@ -5,6 +5,7 @@
   "requires": true,
   "packages": {
     "": {
+      "name": "mattermost-webapp",
```

Comment:
@michelengelen package-lock.json tricked you again, @M-ZubairAhmed are we planning to add this?

Reply:
Argh! 😂

Reply:
yeah lets keep it in this PR, it was removed from a PR from our own team 😛 missed that.